<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DebugWidget</class>
 <widget class="QWidget" name="DebugWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>304</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>12</number>
   </property>
   <property name="leftMargin">
    <number>18</number>
   </property>
   <property name="rightMargin">
    <number>18</number>
   </property>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>类型</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QRadioButton" name="privateRadio">
        <property name="text">
         <string>私聊</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="groupRadio">
        <property name="text">
         <string>群聊</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="noticeRadio">
        <property name="text">
         <string>通知</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>内容</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QRadioButton" name="msgRadio">
        <property name="text">
         <string>消息</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="imgRadio">
        <property name="text">
         <string>图片</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="fileRadio">
        <property name="text">
         <string>文件</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QTextEdit" name="messageEdit"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="topMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QCheckBox" name="showWidgetBorderCheck">
       <property name="text">
        <string>显示控件边界</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="InteractiveButtonBase" name="sendButton">
       <property name="text">
        <string>发送</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>InteractiveButtonBase</class>
   <extends>QPushButton</extends>
   <header>interactivebuttonbase.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
